tests/coafileTest.py

```diff
@@ -0,0 +1,27 @@
+import unittest
+import re
+from coalib.misc.Shell import run_shell_command
+from coalib.output.printers.LogPrinter import LogPrinter
+from pyprint.ConsolePrinter import ConsolePrinter
+from coalib.settings.ConfigurationGathering import load_config_file
+from coalib.output.printers.LOG_LEVEL import LOG_LEVEL
+
+
+class coafileTest(unittest.TestCase):
+
+    def setUp(self):
+        self.logprinter = LogPrinter(ConsolePrinter(), LOG_LEVEL.ERROR)
```

Comment:
Ok, this function still needs to be fully removed. I think this is ok until we get that